proxy/truffle-config.js

```diff
@@ -1,11 +1,19 @@
 require('dotenv').config();
 require("ts-node/register");
 const Web3 = require('web3');
-let hdwalletProvider = require('truffle-hdwallet-provider');
+let hdwalletProvider = require('@truffle/hdwallet-provider');
 let schainName = process.env.CHAIN_NAME_SCHAIN;
 let schainRpcUrl = process.env.URL_W3_S_CHAIN;
 let mainnetRpcUrl = process.env.URL_W3_MAIN_NET;
 
+console.log( "URL_W3_MAIN_NET                  =", process.env.URL_W3_MAIN_NET                  );
```

Comment:
I think it could be better to remove printing env variables

Reply:
sure, removed